test/uk/gov/hmrc/merchandiseinbaggage/stubs/CurrencyConversionStub.scala

```diff
@@ -20,22 +20,22 @@ import java.time.LocalDate.now
 import com.github.tomakehurst.wiremock.WireMockServer
 import com.github.tomakehurst.wiremock.client.WireMock.{aResponse, get, urlMatching}
 import com.github.tomakehurst.wiremock.stubbing.StubMapping
+import com.google.inject.Inject
 import play.api.http.Status.OK
 import play.api.libs.json.Json
 import uk.gov.hmrc.merchandiseinbaggage.CoreTestData
-import uk.gov.hmrc.merchandiseinbaggage.config.CurrencyConversionConfiguration
+import uk.gov.hmrc.merchandiseinbaggage.config.AppConfig
 import uk.gov.hmrc.merchandiseinbaggage.model.api.ConversionRatePeriod
 
-object CurrencyConversionStub extends CurrencyConversionConfiguration with CoreTestData {
+class CurrencyConversionStub @Inject() (config: AppConfig) extends CoreTestData {
 
   def givenCurrencyConversion(code: String = "EUR")(implicit server: WireMockServer): StubMapping =
     server.stubFor(
-      get(urlMatching(s"${currencyConversionConf.currencyConversionUrl}(.*)?cc=$code"))
+      get(urlMatching(s"${config.currencyConversionConf.currencyConversionUrl}(.*)?cc=$code"))
```

Comment:
This can just be `get(urlMatching(s"/currency-conversion/rates/(.*)?cc=$code"))`